src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/message.ts

```diff
@@ -37,20 +27,9 @@ export const useMessageStore = defineStore("message", {
     messageById(id: string): Components.Schemas.Communication | undefined {
       return this.messages?.find((message) => message.id === id);
     },
-    async fetchMessages() {
-      this.messages = (await getMessages()).data;
-    },
+
     async markMessageAsRead(messageId: string) {
```

Comment:
We can probably just call this API method directly and remove from store?

Reply:
Yup, Done..